lib/queries/organizationQueries.ts

```
@@ -0,0 +1,434 @@
+
+type OrganizationGlobalProblemWithProblem = Prisma.OrganizationGlobalProblemGetPayload<{
+  include: { globalProblem: true }
+}>
+
+type OrganizationGlobalSolutionWithSolution = Prisma.OrganizationGlobalSolutionGetPayload<{
+  include: { globalSolution: true }
+}>
+
+export interface OrganizationRelationships {
+  members: OrganizationMembershipWithPerson[]
+  products: Prisma.ProductGetPayload<{}>[]
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)